setting-up/workload-onboarding/container/index.md

```diff
@@ -77,7 +46,7 @@ docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
 5. `--`: Separate Bacalhau parameters from the command to be executed inside the container
 6. `sh -c 'uname -a && echo "Hello from Docker Bacalhau!"'`: The command executed inside the container
 
-Let's have a look at the command execution in the terminal:
+The command execution in the terminal is similar to:
```

Comment:
The former feels friendlier.

Reply:
Followed the word standard writing guidelines. https://developers.google.com/style/accessibility